packages/core/src/modules/credentials/protocol/v2/V2CredentialService.ts

```diff
@@ -1082,7 +1101,7 @@ export class V2CredentialService extends CredentialService {
       if (msg.format.includes('indy')) {
         formats.push(this.getFormatService(CredentialFormatType.Indy))
       } else if (msg.format.includes('aries')) {
-        // todo
+        formats.push(this.getFormatService(CredentialFormatType.JsonLd))
       } else {
         throw new AriesFrameworkError(`Unknown Message Format: ${msg.format}`)
       }
```

Comment:
We should remove this method. It requires the credential service to be aware of the different format services. In the proof PR this is solved by adding a method to the format service asking whether it supporst the format (just like you have the `getAttachment`on the format service that knows which format identifier to look for. This makes it possible to add completely new formats in the future without needing to modfiy the credential service (and would even allow for people to add them dynamically by implementing the format service interface) this would prevent that from working.